src/SecretsApi/SecretsApi.php

```diff
@@ -37,7 +37,7 @@ private function getBaseURI()
         }
         // If host is still not set, use the default host.
         if (!$host) {
-            $host = 'api.pantheon.io';
+            $host = 'terminus.pantheon.io';
```

Comment:
I am ambivalent about this. `api.pantheon.io` specifically means the PantheonAPI service. `terminus.pantheon.io` historically has indicated the service that authenticates for Terminus and forwards to ygg rest, which used to point to Hermes and now points to PantheonAPI. It's hard to predict the future, but I am wondering if there is some value in keeping these separate. Did you have any motivation in making this change, e.g. to perhaps decommission api.pantheon.io someday?

Reply:
I agree with this change.  Past-Pantheon choosing to have a separate hostname specifically for terminus enabled us to change where customer traffic goes without requiring them to install an update.  Continuing to use a hostname just for terminus keeps with this.

It's difficult to imagine why terminus.pantheon.io would be pointed somewhere we didn't want terminus traffic to go.